projects/storefrontapp-e2e-cypress/cypress/helpers/cart-validation.ts

```diff
@@ -44,7 +44,7 @@ export function addMultipleProductsToCart(products) {
 
 export function removeItemAndCheckCartEntriesNumber(
   product,
-  expectedCartLength = 0
```

Comment:
removed as I don't think this should be here. The default value is not used by anything referencing it and I think the caller should have to input their expected length.